src/sagemaker/serve/utils/telemetry_logger.py

```diff
@@ -63,11 +64,15 @@ def wrapper(self, *args, **kwargs):
                 f"{func_name}"
                 f"&x-modelServer={MODEL_SERVER_TO_CODE[str(self.model_server)]}"
                 f"&x-imageTag={image_uri_tail}"
+                f"&x-pySdkVersion={pysdk_version()}"
```

Comment:
Create the constant `SDK_VERSION` and then reuse it wherever needed.

Reply:
Will use the one from here [sagemaker-python-sdk/src/sagemaker/user_agent.py](https://github.com/aws/sagemaker-python-sdk/blob/87d661a8caff9c0671bce183000be6133d9103b9/src/sagemaker/user_agent.py#L21)